packages/components/src/components/custom-select-list/custom-select-list.lite.tsx

```diff
@@ -8,16 +8,15 @@ useDefaultProps<DBCustomSelectListProps>({});
 
 export default function DBCustomSelectList(props: DBCustomSelectListProps) {
 	// This is used as forwardRef
-	const _ref = useRef<HTMLDivElement | any>(null);
+	const _ref = useRef<HTMLFieldSetElement | any>(null);
 
 	return (
-		<div
-			role={props.multiple ? 'group' : 'radiogroup'}
-			aria-label={props.label}
+		<fieldset
 			ref={_ref}
 			id={props.id}
 			className={cls('db-custom-select-list', props.className)}>
+			{props.label && <legend>{props.label}</legend>}
```

Comment:
```suggestion
			className={cls('db-custom-select-list', props.className)}
			role={props.multiple ? undefined : 'radiogroup'}
			aria-label={groupLabel ? undefined : props.ariaListLabel ?? props.label}>
			{groupLabel && <legend className="visually-hidden">{groupLabel}</legend>}
```

Keep the accessible name/role: use ariaListLabel ?? label for the <legend>, so groups without a visible label still get a name. Also, <fieldset> is role="group" by default. Add role="radiogroup" for single-select when children aren’t native radios to preserve semantics.